crates/example-types/src/block_types.rs

```diff
@@ -349,7 +351,7 @@ impl<
     }
 
     fn get_auction_results(&self) -> Option<TYPES::AuctionResult> {
-        unimplemented!()
+        Some(TYPES::AuctionResult { urls: vec![] })
```

Comment:
Can we make an issue to update this once the simple builder supports the marketplace?

Reply:
We should also enable transaction generation and checking that blocks contain transactions in tests when simple builder is updated, so that the test check the builder interaction as well.

Reply:
Done! https://github.com/EspressoSystems/HotShot/issues/3583